synapse/federation/federation_server.py

```diff
@@ -221,6 +230,36 @@ def on_event_auth(self, origin, room_id, event_id):
             "auth_chain": [a.get_pdu_json(time_now) for a in auth_pdus],
         }))
 
+    @defer.inlineCallbacks
+    def on_query_auth_request(self, origin, content, event_id):
```

Comment:
This could use a little bit of doc string to explain what "rejects" and "missing" mean.